.github/workflows/coverage-reporter.yml

```diff
@@ -0,0 +1,31 @@
+---
+## Workflow to process coverage report
+## For now the coverage is not reported. This workflow is just here as preparation for
+## https://github.com/elastic/observability-robots/issues/1546
+name: coverage-reporter
+on:
+  workflow_run:
+    workflows:
+      - test
+    types:
+      - completed
+
```

Comment:
Can you add the github token [permissions](https://docs.github.com/en/actions/security-guides/automatic-token-authentication#permissions-for-the-github_token)?

Reply:
https://github.com/elastic/apm-agent-ruby/pull/1366/commits/74711d147756d66723046fddcc62b5ce8ede2cca